socket.go

```diff
@@ -202,27 +185,23 @@ func shouldAcceptNewConnectionHandler(listenerPtr, connPtr, devicePtr unsafe.Poi
 //
 // see: https://developer.apple.com/documentation/virtualization/vzvirtiosocketconnection?language=objc
 type VirtioSocketConnection struct {
-	sourcePort      uint32
-	destinationPort uint32
-	fileDescriptor  uintptr
-	file            *os.File
-	laddr           net.Addr // local
-	raddr           net.Addr // remote
+	net.Conn
+	laddr *Addr // local
+	raddr *Addr // remote
 }
 
 var _ net.Conn = (*VirtioSocketConnection)(nil)
```

Comment:
Remove this one
```suggestion
```

Reply:
This is the only change in the branch I just force-pushed.